Directory.Build.props

```diff
@@ -1,21 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <PropertyGroup>
-    <NoWarn>$(NoWarn);NU5125;NU1701</NoWarn>
-  </PropertyGroup>
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
-  <!-- Nuget settings -->
   <PropertyGroup>
-    <!-- Use Directory.Packages.props for nuget package versions -->
-    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <Copyright>$(CopyrightMicrosoft)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <LangVersion>latest</LangVersion>
```

Comment:
Just curious, why? We use latest or preview in many repositories in the stack.

Reply:
I also prefer latest. Because of its size and relative vijsibility, this repo should be designed to avoid stagnating as much as possible.

Reply:
In this case let's just use the most recent fixed version supported by the compiler. I believe that's 11 here, but it might be 10. Feel free to follow-up offline.

Reply:
Updated.